test/e2e/run_test.go

```diff
@@ -1168,6 +1168,28 @@ USER mail`, BB)
 		Expect(session.OutputToString()).To(ContainSubstring("data"))
 	})
 
+	It("podman run --volumes-from flag mount conflicts with image volume", func() {
+		volPathOnHost := filepath.Join(podmanTest.TempDir, "myvol")
+		err := os.MkdirAll(volPathOnHost, 0755)
+		Expect(err).ToNot(HaveOccurred())
+
+		imgName := "testimg"
+		volPath := "/myvol/mypath"
+		dockerfile := fmt.Sprintf(`FROM %s
+RUN mkdir -p %s
+VOLUME %s`, ALPINE, volPath, volPath)
+		podmanTest.BuildImage(dockerfile, imgName, "false")
+
+		ctr1 := "ctr1"
+		run1 := podmanTest.Podman([]string{"run", "-d", "-v", fmt.Sprintf("%s:%s:z", volPathOnHost, volPath), "--name", ctr1, ALPINE, "top"})
+		run1.WaitWithDefaultTimeout()
+		Expect(run1).Should(Exit(0))
+
+		run2 := podmanTest.Podman([]string{"run", "--volumes-from", ctr1, imgName, "ls", volPath})
+		run2.WaitWithDefaultTimeout()
+		Expect(run2).Should(Exit(0))
```

Comment:
shouldn't this at least check that we got the volume from the other container?
I mean let ctr1 one create a file in the volume that check for it here.

Because right now this could pass for different reasons.